sdk/confidentialledger/azure-security-confidentialledger/src/test/java/com/azure/security/confidentialledger/ConfidentialLedgerClientTestBase.java

```diff
@@ -1,104 +1,67 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
```

Comment:
Sameeksha pointed this out as well, but to remove:
`// Code generated by Microsoft (R) AutoRest Code Generator.` from files that were handwritten... unless this was auto-generated?

Reply:
These were autogenerated and then modified (by me) to make sure they were functional.